superduperdb/components/vector_index.py

```diff
@@ -168,6 +170,23 @@ def dimensions(self) -> int:
             return shape[-1]
         raise ValueError('Couldn\'t get shape of model outputs from model encoder')
 
+    @override
+    def schedule_jobs(
+        self,
+        db: Datalayer,
+        dependencies: t.Sequence['Job'] = (),
+    ) -> t.Sequence[t.Any]:
+        """
+        Schedule jobs for the listener
+
+        :param database: The DB instance to process
+        :param dependencies: A list of dependencies
+        :param verbose: Whether to print verbose output
+        """
+        # This will trigger the backfilling of
```

Comment:
This is incorrect - we should return a job. Please see the other `schedule_job`. E.g. `Listener`

Reply:
done